agent_server.py

```
@@ -690,12 +705,155 @@ async def _do_analyze_and_plan(messages: list[dict[str, Any]], lanlan_name: Opti
+                    if _reg:
+                        _reg["status"] = up_terminal
+                        _reg["result"] = up_result.result
+                        if not up_result.success and up_result.error:
+                            _reg["error"] = str(up_result.error)[:500]
+                    run_data = up_result.result.get("run_data") if isinstance(up_result.result, dict) else None
+                    detail = str(run_data)[:500] if run_data else ""
+                    if up_result.success:
+                        logger.info(f"[TaskExecutor] ✅ UserPlugin completed: {plugin_id}")
+                        summary = f'插件任务 "{plugin_id}" 已完成'
+                        if detail:
+                            summary = f'插件任务 "{plugin_id}" 已完成：{detail}'
```

Ruff (0.15.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)